models/tasks.js

```diff
@@ -5,7 +5,7 @@ const dependencyModel = firestore.collection("taskDependencies");
 const userUtils = require("../utils/users");
 const { fromFirestoreData, toFirestoreData, buildTasks } = require("../utils/tasks");
 const { TASK_TYPE, TASK_STATUS, TASK_STATUS_OLD, TASK_SIZE } = require("../constants/tasks");
-const { IN_PROGRESS, BLOCKED, SMOKE_TESTING, COMPLETED } = TASK_STATUS;
+const { IN_PROGRESS, BLOCKED, SMOKE_TESTING, DONE } = TASK_STATUS;
```

Comment:
will this change affect the frontend sites which are currently using the old statuses ? will it not cause breaking changes in the frontend repos !

Reply:
The change in tasks constants file could potentially break the status page. As the status page renders tasks with status as 'COMPLETED' and I have removed it from constants (as we are updating all tasks with 'COMPLETED' status to 'DONE').

Reply:
In that case it should be handled in those sites as well so that they don't start breaking.